empress/support_files/js/util.js

```diff
@@ -126,25 +141,58 @@ define(["underscore"], function (_) {
     }
 
     /**
-     * Display a message in toast.
+     * Display a message in a toast element.
      *
-     * @param {string} msg - message to display
-     * @param {number} duration - milliseconds to keep toast visible
+     * @param {String} msg - message to display
+     * @param {Number} duration - milliseconds to keep toast visible (optional)
+     *                            If this is undefined, then this will default
+     *                            to 3000 ms (i.e. 3 seconds).
      */
-    function toastMsg(msg) {
-        var duration = 2000,
-            toast = document.getElementById("toast");
+    function toastMsg(msg, duration) {
+        var effectiveDuration = _.isUndefined(duration) ? 3000 : duration;
+        var toast = document.getElementById("toast");
```

Comment:
```suggestion
    function toastMsg(msg, duration=3000) {
        var toast = document.getElementById("toast");
```

Not a big deal, I just think it looks a little cleaner.

Reply:
Agreed that this does seem nicer. It looks like default parameters don't work on Internet Explorer, though (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#Browser_compatibility)). @ElDeveloper we're not officially supporting IE, right?

Relevant to #222...

Reply:
No, I don't think we are supporting IE. I know multiple functions (such as the modifications Antonio made to bp-tree) use default args. I also have some recollection that some other code we used on is not support in IE (although I don't remember what that was).